tensor_comprehensions/library/layers.yaml

```diff
@@ -41,6 +41,12 @@
       output(i, j) +=! A(i, kk) * B(kk, j)
     }
 
```

Comment:
the new line is not needed and will break yaml parsing. see line 146-157 as an example :)